src/inputmethodkit.cs

```diff
@@ -51,14 +47,16 @@ partial interface IMKServer : IMKServerProxy {
 		NSBundle Bundle { get; }
 
 		[Lion, Export ("paletteWillTerminate")]
-		bool PaletteWillTerminate { get; }
+		bool PaletteWillTerminate ();
 
 		[Lion, Export ("lastKeyEventWasDeadKey")]
 		bool LastKeyEventWasDeadKey { get; }
 	}
 
-	[Category, BaseType (typeof (NSObject))]
-	partial interface IMKServerInput_NSObject {
+	interface IIMKServerInput { }
+
+	[Protocol]
+	partial interface IMKServerInput {
```

Comment:
You want `[Protocol (IsInformal = true)]` here

Because: https://developer.apple.com/reference/inputmethodkit/imkserverinput?language=objc
>IMKServer​Input is an informal protocol that defines methods for receiving text events. This is intentionally not a formal protocol because there are three ways to receive events.